views/index.html

```diff
@@ -235,9 +235,12 @@
 <ol>
   <li>From the Web Console homepage, navigate to your project</li>
   <li>Click on Browse &gt; Builds</li>
-  <li>From the view for your Build click on the button to copy your GitHub webhook</li>
-  <li>Navigate to your repository on GitHub and click on repository settings &gt; webhooks</li>
-  <li>Paste your webhook URL provided by OpenShift &mdash; that's it!</li>
+  <li>Click the nodejs-example link that represents the BuildConfig for this example application</li>
```

Comment:
The actual build config name might not be nodejs-example depending on how you created the app.